doc/docs/2.3.x/concepts/pipeline-concepts/pipeline/cron.md

```diff
@@ -1,69 +1,79 @@
+Cron pipelines are great for tasks like:
 
-| Parameter  | Description  |
+- Scraping websites
+- Making API calls
+- Querying a database
+- Retrieving a file from a location accessible through an S3 protocol or a File Transfer Protocol (FTP).
+
+### Behavior 
+
+When you create a Cron pipeline, Pachyderm creates a new input data repository that corresponds to the `cron` input Then, Pachyderm automatically commits a timestamp file to the `cron` input repository at your determined interval, which triggers the pipeline.  
+
+By default, each cron trigger adds a new tick file to the cron input repository, accumulating more datums over time. 
+Optionally, you can set the overwrite flag to `true` to overwrite the timestamp file on each tick. To learn more about overwriting commits in Pachyderm, see [Datum processing](../datum/index.md).
+
+### Required Parameters 
+
+At minimum, a Cron pipeline must include all of the following parameters:
+
+| Parameters  | Description  |
 | ---------- | ------------ |
 | `"name"`   | A descriptive name of the cron pipeline. |
-| `"spec"`   | An interval between scheduled cron jobs. You can specify any value that is <br> formatted according to [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt){target=_blank}. <br> For example, if you set `*/10 * * * *`, the pipeline runs every ten minutes. |
-
-## Example of a Cron Pipeline
-
-For example, you want to query a database every ten seconds and update your
-dataset with the new data every time the pipeline is triggered. The following
-pipeline extract illustrates how you can specify this configuration.
-
-!!! example
-
-    ```json
-      "input": {
-        "cron": {
-          "name": "tick",
-          "spec": "@every 10s"
-        }
-      }
-    ```
-
-When you create this pipeline, Pachyderm creates a new input data repository
-that corresponds to the `cron` input. Then, Pachyderm automatically commits
-a timestamp file to the `cron` input repository every ten seconds, which
-triggers the pipeline.
-
-![alt tag](../../../assets/images/cron1.png)
-
-The pipeline runs every ten seconds, queries the database and updates its
-output. By default, each cron trigger adds a new tick file to the cron input
-repository, accumulating more datums over time. This behavior works for some
-pipelines. For others, you might want each tick file to overwrite the
-previous one. You can set the overwrite flag to true to overwrite the
-timestamp file on each tick. To learn more about overwriting commits in
-Pachyderm, see [Datum processing](../datum/index.md).
-
-!!! example
-
-    ```json
-      "input": {
-        "cron": {
-          "name": "tick",
-          "spec": "@every 10s",
-          "overwrite": true
-        }
-      }
-    ```
+| `"spec"`   | The interval between scheduled cron jobs; accepts [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) inputs as well as `never`, `@hourly`, `@daily`, `@weekly`, `@monthly`, `@yearly`, `@reboot`, and `@every ...`|
```

Comment:
What does `@reboot` do? The others are more obvious, but maybe we should add a little note on `@reboot` and also what can be used with `@every`.
For example, if I want it to be every 3 months, what would that be? `@every 3m` would be minutes, not months, right?

Reply:
Found some direct links to those supported formats in the cron.go pkg!